### PR TITLE
Authentication

### PR DESCRIPTION
**Pull request by timhillu**
Adds support for authentication, begins support for admin-level-checking.

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -41,11 +43,47 @@ class App extends Component {
...
     base.removeBinding(this.congregationsRef);
     base.removeBinding(this.sermonsRef);
+    base.removeBinding(this.adminRef);
+  }
+
+  loginCheck() {
+    const self = this;
+    var authHandler = function(user) {
+      if(user) {
+//        console.log('You are logged in right now!');
```
Comment (smcadams86) [inline src/App.js]: can you delete the commented lines ?

File: src/modules/Congregations.js [hunk 2]
```diff
@@ -12,6 +13,21 @@ class Congregations extends React.Component {
     this.handleChange = this.handleChange.bind(this);
   }
 
+  login = () => {
+    var authHandler = function(error, user) {
```
Comment (smcadams86) [inline src/modules/Congregations.js]: delete this auth handler if it was only for debugging purposes

Comment (timhillu) [inline src/modules/Congregations.js]: It's called when the login button is pressed.

Comment (smcadams86) [inline src/modules/Congregations.js]: yes, but it's not actually doing anything

File: src/modules/Sermons.js [hunk 3]
```diff
@@ -3,7 +3,6 @@ import React from 'react'
 class Sermons extends React.Component {
   render() {
     // TODO
-    return <div>Sermons List</div>
```
Comment (smcadams86) [inline src/modules/Sermons.js]: why did this get deleted? I don't think this page will render without returning some content?

Comment (timhillu) [inline src/modules/Sermons.js]: Probably disappeared when I was manually merging all of those changes.  It still renders, just throws a warning.

Added back in.

File: src/modules/Congregations.js (hunk 2 above)
Comment (timhillu) [inline src/modules/Congregations.js]: True enough, deleted it.

**Comment by smcadams86** [comment]
nice work!

**Pull request merged**